cli/flox-rust-sdk/src/models/env_registry.rs

```diff
@@ -30,6 +32,10 @@ pub enum EnvRegistryError {
     WriteEnvironmentRegistry(#[source] SerializeError),
     #[error("no registry found")]
     NoEnvRegistry,
+    #[error(transparent)]
+    Git(#[from] GitCommandError),
+    #[error(transparent)]
+    FloxMeta(#[from] FloxMetaError),
```

Comment:
I rage quit about here, so feel free to suggest alternatives.

Reply:
Removed `GitCommandError` because it wasn't used and snuck through from a previous refactor.